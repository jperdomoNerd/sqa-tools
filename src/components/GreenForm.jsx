const GreenForm = () => {
  return (
    <form className="form form--green">
      <h3 className="form__title">Parameters in use:</h3>
      <div className="form__input-container">
        <label className="form__label">apikey:</label>
        <input className="form__input" type="text" />
      </div>
      <div className="form__input-container">
        <label className="form__label">aeskey:</label>
        <input className="form__input" type="text" />
      </div>
    </form>
  );
}