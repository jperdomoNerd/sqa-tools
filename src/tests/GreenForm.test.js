import renderer from 'react-test-renderer';
import GreenForm from '../components/GreenForm';

describe('GreenForm Component', () => {
  
  it('<GreenForm /> renders correctly', () => {
    const tree = renderer.create(<GreenForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
