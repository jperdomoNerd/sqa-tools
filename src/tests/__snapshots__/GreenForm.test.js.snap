// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreenForm Component <GreenForm /> renders correctly 1`] = `
<form
  className="form form--green"
>
  <h3
    className="form__title"
  >
    Parameters in use:
  </h3>
  <div
    className="form__input-container"
  >
    <label
      className="form__label"
    >
      apikey:
    </label>
    <input
      className="form__input"
      type="text"
    />
  </div>
  <div
    className="form__input-container"
  >
    <label
      className="form__label"
    >
      aeskey:
    </label>
    <input
      className="form__input"
      type="text"
    />
  </div>
</form>
`;
